projects/legacy/components/input-tag/input-tag.style.less

```diff
@@ -82,6 +82,10 @@
     :host[data-size='l']:not(._label-outside) & {
         padding: 0;
     }
+
+    & > tui-tag:has(+ .t-input-wrapper) {
```

Comment:
No need for `:has` stuff, just the `:last-child` — we don't want a single tag to only take up 80% of the input if it is alone on the row, we only want the last one which is right next to input

Reply:
I don't understand how it should work
<img width="541" alt="image" src="https://github.com/user-attachments/assets/115594d9-8665-4358-a9b7-288063bb9123">

<img width="287" alt="image" src="https://github.com/user-attachments/assets/8e47fe62-9f05-4072-91bd-3360b594f5f0">

Reply:
It's not `:last-child`, apparently:
<img width="1096" alt="image" src="https://github.com/user-attachments/assets/db1fe356-011a-4174-a968-8ff1faa034ee">
`:last-of-type` should work.

Reply:
fixed